localisation/00_wc_economy.csv

```diff
@@ -0,0 +1,19 @@
+#CODE;ENGLISH;FRENCH;GERMAN;;SPANISH;;;;;;;;;x
+borrow_money;Borrow 300�;;Leiht Euch 300�;;;;;;;;;;x;
+borrow_money_desc;You can borrow 300� from some rich ruler or moneylenders.;;Ihr k�nnt Euch 300� von einem reichen Herrscher oder Geldverleiher leihen.;;;;;;;;;;x;
+repay_money;Repay 360� Loan;;Zahlt 360� Darlehen zur�ck;;;;;;;;;;;x
+repay_money_desc;You repay the gold you owe.;;Ihr Zahlt das Darlehen zur�ck, das Ihr schuldig seid.;;;;;;;;;;;x
+debt_360_gold;Owes 360�;;360� Darlehen;;;;;;;;;;;x
+interest_payment_360_gold;Receives Interest Payments;;Erh�lt Einkommen aus Zinsen;;;;;;;;;;;x
+EVTDESC_WCECO_100;You ask your servants to look for moneylenders who can borrow gold for your goals.\n\nWithin the same day, they return with news.;;Ihr bittet Eure Diener, nach Geldverleihern zu suchen, die Euch Gold f�r Eure Ziele leihen k�nnen.\n\nIm Laufe des Tages kehren sie mit Neuigkeiten zur�ck.;;;;;;;;;;x;
+EVTOPTA_WCECO_100;[target_moneylender.GetFirstName] you say... deliver [target_moneylender.GetHerHim] my letter!;;[target_moneylender.GetFirstName] sagt ihr... �berbringt [target_moneylender.Getihmihr] meinen Brief!;;;;;;;;;;x;
+EVTOPTB_WCECO_100;The local community is giving loans... interesting...;;Die hiesige Gemeinde gew�hrt Darlehen... interessant...;;;;;;;;;;x;
+EVTDESC_WCECO_110;[Root.PrimaryTitle.GetFOA],\nlegends are told about your wealth. May you allow me to borrow a fraction of your gold. We will return all the gold and more, to the last coin.\n\nSigned,\n[target_borrower.GetTitledName];;[Root.PrimaryTitle.GetFOA],\nes werden Legenden �ber Euren Reichtum verbreitet. D�rfte ich mir einen Teil Eures Goldes leihen? Wir werden das Gold bis zur letzten M�nze zur�ckzahlen und mehr.\n\nGezeichnet,\n[target_borrowrower.GetTitledName];;;;;;;;;;x;
```

Comment:
> legends are told about your wealth

600 wealth
@
legends

When my char got his first 600, it received 3 borrowing requests in a single month. I accepted them all and my char become bankrupt. Not really a legendary wealth, eh?

How about change that part?

Reply:
> How about change that part?

@MemeWizard are you sure we want to change it? The AI shouldn't be able to accept them all.

Reply:
I interpreted this line as coming off as schmoozing your lender, not so much telling how it is.

Reply:
> I interpreted this line as coming off as schmoozing your lender, not so much telling how it is.

Dis?

> legends are told about your wealth

Ye, that's what I meant. Oh, @MemeWizard, so you want to change the localization?

Reply:
> Oh, @MemeWizard, so you want to change the localization?

Yep. Maybe change it to actual [compliment](https://ck2.paradoxwikis.com/Trait_modding#Compliments)?

Reply:
> Yep. Maybe change it to actual [compliment](https://ck2.paradoxwikis.com/Trait_modding#Compliments)?

@MemeWizard, please, no. They're big dumb.
P.S. I don't know why he says "god king" but others don't look better, just adjective + noun.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79785835-998f8500-8355-11ea-9fe6-53677b9a5d40.png)

</details>

Reply:
> I don't know why he says "god king"

Because you used wrong scopes.

> others don't look better, just adjective + noun

Every text is just a set of the same characters in different order.

> They're big dumb.

Personally, I think that random adjective + random noun is much better than the current static version.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/79830023-36314180-83ad-11ea-81f0-07577a535a19.png)
![image](https://user-images.githubusercontent.com/46576860/79830042-42b59a00-83ad-11ea-98b0-85272e6bf7f0.png)
![image](https://user-images.githubusercontent.com/46576860/79830054-4ba66b80-83ad-11ea-8092-b3e5f0c63c89.png)
![image](https://user-images.githubusercontent.com/46576860/79830129-71337500-83ad-11ea-958b-9af48ce2aba0.png)
![image](https://user-images.githubusercontent.com/46576860/79830084-5bbe4b00-83ad-11ea-8abd-9b4f17126c87.png)
![image](https://user-images.githubusercontent.com/46576860/79830109-6842a380-83ad-11ea-931d-250d7e771a0e.png)
![image](https://user-images.githubusercontent.com/46576860/79830171-8f00da00-83ad-11ea-8371-a4649bcea485.png)
![image](https://user-images.githubusercontent.com/46576860/79830264-bd7eb500-83ad-11ea-8148-bf66ae611a40.png)

</details>

Reply:
I like random adjective/noun combo.
I like `strong god king`

Reply:
If anything, this is what I used
```suggestion
EVTDESC_WCECO_110;To the [GetFromRootCompliment], [Root.GetFirstNameWithNick].\nMay you allow me to borrow a fraction of your gold. We will return all the gold and more, to the last coin.\n\nSigned,\n[target_borrower.GetTitledName];;[Root.PrimaryTitle.GetFOA],\nes werden Legenden �ber Euren Reichtum verbreitet. D�rfte ich mir einen Teil Eures Goldes leihen? Wir werden das Gold bis zur letzten M�nze zur�ckzahlen und mehr.\n\nGezeichnet,\n[target_borrowrower.GetTitledName];;;;;;;;;;x;
```

Reply:
Done in 7fe692579d26b1be4f82aea52b5fb075ef431f76.